mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping

```diff
@@ -44,24 +49,104 @@ CLASS net/minecraft/class_473 net/minecraft/client/gui/screen/ingame/BookEditScr
 	METHOD method_27576 getPageContent ()Lnet/minecraft/class_473$class_5233;
 	METHOD method_27577 invalidatePageContent ()V
 	METHOD method_27578 createPageContent ()Lnet/minecraft/class_473$class_5233;
+	METHOD method_27580 moveCursorLine (I)V
+		ARG 1 offset
+	METHOD method_27581 drawCursor (Lnet/minecraft/class_4587;Lnet/minecraft/class_473$class_5234;Z)V
+		ARG 1 matrices
+		ARG 2 position
+		ARG 3 haveSelection
+	METHOD method_27582 screenToBook (Lnet/minecraft/class_473$class_5234;)Lnet/minecraft/class_473$class_5234;
+		COMMENT Converts a position as a screen coordinate to a position relative to the book UI
+		ARG 1 position
+	METHOD method_27583 createSelectionBoundingBox (Lnet/minecraft/class_473$class_5234;Lnet/minecraft/class_473$class_5234;)Lnet/minecraft/class_768;
+		ARG 1 topLeft
+		ARG 2 bottomRight
+	METHOD method_27584 setClipboard (Ljava/lang/String;)V
+		ARG 1 text
+	METHOD method_27585 createSelectionBoundingBox (Ljava/lang/String;Lnet/minecraft/class_5225;IIII)Lnet/minecraft/class_768;
+		ARG 1 text
+		ARG 2 textHandler
+		ARG 3 fromOffset
+		ARG 4 toOffset
+		ARG 5 lineY
+		ARG 6 lineStart
+	METHOD method_27586 syntheticSplitLine (Lorg/apache/commons/lang3/mutable/MutableInt;Ljava/lang/String;Lorg/apache/commons/lang3/mutable/MutableBoolean;Lit/unimi/dsi/fastutil/ints/IntList;Ljava/util/List;Lnet/minecraft/class_2583;II)V
```

Comment:
synthetic methods should not have names. IDK how you named these?

Reply:
I'm pretty sure they showed up in Enigma?

Reply:
Ill take a look. It should only allow you to name the params.